packages/ui/src/documents/ListDocuments.tsx

```diff
@@ -185,7 +192,13 @@ export const ListDocuments: FunctionComponent<Props> = (props: Props) => {
               sortable: false,
               size: '36px',
               render: datum => {
-                return canLoadDocument(datum) ? <Box><FormNext /></Box> : <></>;
+                return canLoadDocument(datum) ? (
+                  <Box>
+                    <FormNext />
+                  </Box>
+                ) : (
+                  <></>
```

Comment:
Would `null` here rather than an empty fragment resolve to the same?

Reply:
Yes, absolutely. Again, this was my editor doing a bit magic, sorry :(